### DISCUSSION TITLE
[29/01/24] - Daily Work Notes:

### DISCUSSION THREAD

**Discussion by Nintails1** [Nintails-TF/ncr-agile-wombo#147] [Meeting Notes]
- **A big problem** that if we look at #146 - a user story from NCR wants 2FA or SSO. We currently don't have the architecture for regular users logging in. As it isn't super important to improve user experience. But it can provide large amounts of value for the client. As if we create user accounts, we can get personalised analytic data from them, which helps other user stories.
   -  e.g. (#145, #142, #12).

**Comment by lewissimmonds** [comment]
This could be solved using cookies and session storage.
